srl/src/main/java/edu/illinois/cs/cogcomp/srl/utilities/WeightVectorUtils.java

```diff
@@ -0,0 +1,134 @@
+/**
```

Comment:
`core-utils` doesn't have `sl` as dependency (and vice versa). Maybe we keep it here for the time being?